relay/workflow.py

```diff
@@ -457,15 +476,15 @@ def get_handler(self, event:dict):
 
             elif t == 'wf_prompt_event':
                 if event['type'] == 'started':
-                    h = self.type_handlers.get(('wf_api_prompt_start_event'), None)
+                    h = self.type_handlers.get('wf_api_prompt_start_event', None)
```

Comment:
are these prompt events correct?  I don't see wf_prompt_event in relay_erlang.  and do we set handlers for wf_api_prompt_start_event and _stop_event?

Reply:
Yeah, I'm not seeing those events in ibot source or relay_erlang. I'll remove them.